gapis/api/vulkan/BUILD.bazel

```diff
@@ -63,8 +63,8 @@ go_library(
         "draw_call_mesh.go",
         "externs.go",
         "find_issues.go",
-        "footprint_builder.go",
         "graph_visualization.go",
+        # "footprint_builder.go",
```

Comment:
Can we actually just remove these from the build, rather htan commenting them out.

Reply:
I will pull the stale code dropping part out and make another CL, this one will just be rebased on the new one.